test/pylib/minio_server.py

```diff
@@ -198,13 +204,17 @@ async def start(self):
         os.environ[self.ENV_ADDRESS] = f'{self.address}'
         os.environ[self.ENV_PORT] = f'{self.port}'
         os.environ[self.ENV_BUCKET] = f'{self.bucket_name}'
+        os.environ[self.ENV_ACCESS_KEY] = f'{self.access_key}'
+        os.environ[self.ENV_SECRET_KEY] = f'{self.secret_key}'
 
         try:
             alias = 'local'
             self.log_to_file(f'Configuring access to {self.address}:{self.port}')
             await self.mc('config', 'host', 'rm', alias, ignore_failure=True)
             # wait for the server to be ready when running the first command which should not fail
             await self.mc('config', 'host', 'add', alias, f'http://{self.address}:{self.port}', self.default_user, self.default_pass, timeout=30)
+            self.log_to_file(f'Creating user with key {self.access_key}')
+            await self.mc('admin', 'user', 'add', alias, self.access_key, self.secret_key)
```

Comment:
The various bots that scan for secrets in log files are going to go wild, even though these keys are useless.

But perhaps we shouldn't log them in any case.

Reply:
For example, if someone has real keys in their environment and they get leaked. Not worth the risk.

Reply:
I log access key only, secret key stays in memory only all the time

Reply:
Ah, access key _ID_.